features/comment.feature

```diff
@@ -133,7 +132,7 @@ Feature: Manage WordPress comments through the REST API
     When I try `wp rest comment update 1 --content="Hello World"`
     Then STDERR should contain:
       """
-      Error: Sorry, you can not edit this comment
+      Error: Sorry, you are not allowed to edit this comment.
```

Comment:
These testing updates are already fixed in my PR #92. Please refer to the 3 testing files for a more comprehensive update for WP 4.7+